x-pack/plugin/eql/qa/common/build.gradle

```diff
@@ -5,7 +5,7 @@ dependencies {
   api project(':test:framework')
   api project(path: xpackModule('core'), configuration: 'default')
   api project(path: xpackModule('core'), configuration: 'testArtifacts')
-  api project(path: xpackModule('ql'), configuration: 'testArtifacts')
+  api project(xpackModule('ql:test'))
```

Comment:
I think ideally we would have the logic we want to share from `:ql:test` in a separate test fixture only project. Or is `:qa` already such a fixture project? then we probably want to move the shared fixtures into the main sourceSet of `:qa` instead

Reply:
Ah forget my comment. I was confused. `:qa:test` is that test fixture project. 🤦

Reply:
It is! Should I rename?